data/sql/updates/pending_db_world/rev_1582996192436902600.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1582996192436902600');
+
+-- Update gossip_menu table to allow bigger MenuIDs
+ALTER TABLE `gossip_menu` CHANGE COLUMN `MenuID` `MenuID` INT UNSIGNED NOT NULL DEFAULT 0 FIRST;
+
+-- Zone Stormwind - npc_lady_katrana_prestor gossip menu
+UPDATE npc_text SET TEXT0_0 = '<Lady Prestor glares at you.>', BroadcastTextID0 = '0' WHERE ID = '2693';
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES ('90000', '2693');
+UPDATE `creature_template` SET `gossip_menu_id`='90000' WHERE  `entry`=1749;
```

Comment:
Dont use '' around number where it isn't needed.

Reply:
Why do you add text0_0? 0_1 is for females, and it already exists.
Why do you remove the BroadcastText? There is already one assigned 4990

Reply:
I remember i deleted the broadcast text because it was useless since you can simply have everything on text0_0 and text0_1. Broadcast seems like an extra useless row on the DB that means another query.

Reply:
That query in it itself is not necessary since that exact text exists in text0_1 which is used for female NPC's. Adding it to 0_0 would mean you would add it to a potential male NPC which Lady Prestor is not.